app/api/auth/routes.js

```diff
@@ -82,7 +82,8 @@ export default app => {
 
   app.get('/api/captcha', cors(corsOptions), async (_req, res) => {
     const captcha = svgCaptcha.create({ ignoreChars: '0OoiILluvUV' });
-    const storedCaptcha = await CaptchaModel.save({ text: captcha.text });
+    const text = process.env.DATABASE_NAME !== 'uwazi_e2e' ? captcha.text : '42hf';
```

Comment:
we sure we want to depend on the database_name ? i know the chances are small, but it does not feel right, are we using the same condition for the mail "mock" ?

Reply:
yes, its the same condition than in the mail mock, do not like it either

Reply:
lets go with it and create an tech debt to change both ?